tools/doc-generator/parse/root_blocks.go

```diff
@@ -58,6 +59,11 @@ var (
 			StructType: reflect.TypeOf(frontend.CombinedFrontendConfig{}),
 			Desc:       "The frontend block configures the query-frontend.",
 		},
+		{
+			Name:       "query_scheduler",
+			StructType: reflect.TypeOf(scheduler.Config{}),
+			Desc:       "The query_scheduler configures query scheduler module.",
```

Comment:
Editing to use the naming we agreed on when working on doc.

```suggestion
			Desc:       "The query_scheduler configures query-scheduler component.",
```

Reply:
fixed in https://github.com/grafana/mimir/pull/1623/commits/08b911c1c89331b553c9808fea170b7bf19e8678